BlogTemplate/Pages/New.cshtml.cs

```diff
@@ -59,26 +60,26 @@ public IActionResult OnPostSaveDraft()
 
         private void SavePost(NewPostViewModel newPost, bool publishPost)
         {
+            if (string.IsNullOrEmpty(newPost.Excerpt))
+            {
+                newPost.Excerpt = _excerptGenerator.CreateExcerpt(newPost.Body);
+            }
             Post post = new Post {
-                Title = NewPost.Title,
-                Body = NewPost.Body,
-                Excerpt = NewPost.Excerpt,
-                Slug = _slugGenerator.CreateSlug(NewPost.Title),
+                Title = newPost.Title,
+                Body = newPost.Body,
+                Excerpt = newPost.Excerpt,
+                Slug = _slugGenerator.CreateSlug(newPost.Title),
                 LastModified = DateTimeOffset.Now,
             };
 
-            if (string.IsNullOrEmpty(post.Excerpt))
-            {
-                post.Excerpt = _excerptGenerator.CreateExcerpt(post.Body, 140);
-            }
-
             if (publishPost)
             {
                 post.PubDate = DateTimeOffset.Now;
                 post.IsPublic = true;
             }
 
-            _dataStore.SaveFiles(Request.Form.Files.ToList());
+            //What does Request.Form.Files.ToList() even do?

```

Comment:
If you don't know what this does, don't just comment it out.  Maybe ask the person who wrote it?

Reply:
(you probably just broke a feature with this change)